src/lib/containers/drug_upload_tool/Header.tsx

```diff
@@ -0,0 +1,18 @@
+import * as React from 'react';
+
+export type HeaderProps = {
+  bodyText?: string;
```

Comment:
should probably at least add a comment that this should not be user-defined html (that the bodyText should always be hard-coded) if using dangerouslySetInnerHTML with the value.